platform/web/export/export_plugin.cpp

```diff
@@ -387,7 +387,7 @@ Ref<Texture2D> EditorExportPlatformWeb::get_logo() const {
 bool EditorExportPlatformWeb::has_valid_export_configuration(const Ref<EditorExportPreset> &p_preset, String &r_error, bool &r_missing_templates, bool p_debug) const {
 #ifdef MODULE_MONO_ENABLED
 	// Don't check for additional errors, as this particular error cannot be resolved.
-	r_error += TTR("Exporting to Web is currently not supported in Godot 4 when using C#/.NET. Use Godot 3 to target Web with C#/Mono instead.") + "\n";
+	r_error += TTR("Exporting to Web is currently not supported in Blazium 4 when using C#/.NET. Use Blazium 3 to target Web with C#/Mono instead.") + "\n";
```

Comment:
This should probably be shortened to "Exporting to Web is currently not supported in Blazium 4 when using C#/.NET." since there is no Blazium 3.

Reply:
fixed https://github.com/blazium-engine/blazium/pull/30/commits/25d9fd1d0892b1650c16536d037320d2d6e69271